utils/node_local_test.py

```diff
@@ -1921,6 +1921,47 @@ def test_complex_rename(self):
 
         ofd.close()
 
+    def test_cont_ro(self):
+        """Test access to a read-only container"""
+
+        # Update container ACLs so current user has rta permissions only, the minimum required.
+        rc = run_daos_cmd(self.conf, ['container',
+                                      'update-acl',
+                                      self.pool.id(),
+                                      self.container,
+                                      '--entry',
+                                      'A::{}@:rta'.format(os.getlogin())])
+        print(rc)
+
+        # Assign the container to someone else.
+        rc = run_daos_cmd(self.conf, ['container',
+                                      'set-owner',
+                                      self.pool.id(),
+                                      self.container,
+                                      '--user',
+                                      'root@'])
+        print(rc)
```

Comment:
What if this update-acl and set-owner commands fail for some reason, but dfuse still succeeds below because the default permissions are sufficient?

Reply:
Good question.  In practice I'd hope that if this failed then it would log errors which would show up as a regression, and dfuse would stop logging the permissions error which would show up as a removed warning, so I expect we would notice it but there's definitely a gap here.

Most of the code here asserts rc.returncode, I'll add that here.

Reply:
Thanks!